### PR TITLE
Indent

### LINKED ISSUE #201: indent
http://eslint.org/docs/rules/indent

```
"indent": ["error", 2]
```

4-space indent is reccommended by goodparts http://jslint.com/help.html but going with 2

### PR DESCRIPTION
**Pull request by eliascodes**
* Fixes #201 
* No linting errors
* All tests pass (again)

### CODE REVIEW

**Comment by nelsonic** [comment]
@eliascodes please remove `[WIP]` from PR title if this is ready for review. (_thanks for doing this!_)

File: rules/best_practices.js [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
-  "no-sequences": "error", // https://github.com/dwyl/goodparts/issues/163
-  "no-throw-literal": "error", // https://github.com/dwyl/goodparts/issues/108
-  "no-unused-labels": "error", // https://github.com/dwyl/goodparts/issues/166
-  "no-useless-concat": "error", // https://github.com/dwyl/goodparts/issues/172
-  "no-unmodified-loop-condition": "error", // https://github.com/dwyl/goodparts/issues/178
-  "no-void": "error", // https://github.com/dwyl/goodparts/issues/179
-  "no-with": "error", // https://github.com/dwyl/goodparts/issues/181
-  "vars-on-top": "error", // https://github.com/dwyl/goodparts/issues/182
-  "yoda": ["error", "never", { exceptRange: true }], // https://github.com/dwyl/goodparts/issues/183
+  'accessor-pairs': 'error', // https://github.com/dwyl/goodparts/issues/59
```
Comment (nelsonic) [inline rules/best_practices.js]: did our own `.js` file fail our Linter...?

Comment (eliascodes) [inline rules/best_practices.js]: Yes 😮 . All good now though. The test fixtures are ignored by the linter for obvious reasons.

**Comment by nelsonic** [comment]
@eliascodes looks good. thanks for clarifying the reason for updating all the `"error"` to `2` ... (_backward compatibility for Atom editor etc._)
Merging! 👍 🎉

**Pull request merged**